### PR TITLE
Get Started Doc Updates

### PR DESCRIPTION
**Pull request by mccormickc**

### CODE REVIEW

File: get_started/lb-first-steps.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ In this tutorial, you will learn how to get up and running with Liquibase.
...
 
 ### What if I would rather manually install Liquibase?
-If you choose not to use the Liquibase installer (which comes with tons of example files to help you learn the Liquibase Develop workflow **before** using your own database), then you can view the [Liquibase Installation Documentation](/documentation/installation/index.html) for more information on other ways to download and install Liquibase.
+There are many ways to download and install Liquibase manually depending how you choose to use it. If you decide not to use the Liquibase installer (which comes with tons of example files to help you learn the Liquibase Develop workflow **before** using your own database), then you can view the [Liquibase Installation Documentation](/documentation/installation/index.html) for more information on other ways to download and install Liquibase.
```
Comment (SteveDonie) [inline get_started/lb-first-steps.md]: ...depending *on* how...

File: get_started/lb-first-steps.md [hunk 2]
```diff
@@ -58,26 +58,28 @@ The purpose of these files is to walk you through the Liquibase Developer workfl
...
   </div>
 
 <div class="tile-item" markdown="1">
-Choosing this path means that your changes are defined in XML, JSON or YAML formats. Liquibase will create XML formatted changelogs that define your changeSets, then generate and deploy SQL to your database based on those changeSets. Liquibase will also track all database migrations in your *changelog*. 
+
+Choosing this path means that your changes are defined in XML, JSON, or YAML formats. Liquibase will create XML formatted *changelogs* that define your *changeSets*, then generate and deploy SQL to your database based on those *changeSets*. Liquibase will also track all database migrations in your *changelog*. 
+
 <div class="cta-container" style="margin-left: auto; margin-right: auto; width: 300px; height: 50px">
-<div class="cta cta--block"><a href="/get_started/quickstart_lb.html">Your First Migration: Liquibase Functions ►</a></div></div>
+<div class="cta cta--block"><a href="/get_started/quickstart_lb.html">Your First Migration: Liquibase Commands ►</a></div></div>
```
Comment (SteveDonie) [inline get_started/lb-first-steps.md]: same comment re: commands

File: get_started/lb-first-steps.md [hunk 3]
```diff
@@ -58,26 +58,28 @@ The purpose of these files is to walk you through the Liquibase Developer workfl
 <div class="cta cta--block"><a href="/documentation/workflows/using-the-lb-installer.html">Your First Migration using Sample Liquibase Files ►</a></div></div>
 
 ### **I installed Liquibase Manually**
-If you installed Liquibase manually with a `.zip` or `.tar.gz` file, there are two ways Liquibase allows you to define changes to the database:
+If you installed Liquibase using a manual method, there are two ways Liquibase allows you to define changes to the database:
 
-#### Liquibase Functions
+#### Liquibase Commands
```
Comment (SteveDonie) [inline get_started/lb-first-steps.md]: We need to be careful about changing this terminology. To me, the commands are things like update, rollback. etc. The ChangeTypes are not commands. I could see standardizing on ChangeTypes for this concept.

**Comment by mccormickc** [comment]
Closing 436 in favor of having all changes in 437

**Pull request closed without merging**